test/tests/postprocessors/nearest_node_number/tests

```diff
@@ -0,0 +1,51 @@
+[Tests]
+  issues = '#15749'
+  design = 'NearestNodeNumber.md'
+  [./nearest_node_number_1]
+    type = CSVDiff
+    input = nearest_node_number_1.i
+    csvdiff = nearest_node_number_1_out.csv
+    requirement = "The system shall have the capability to compute the nearest node number to a given point."
+  [../]
+  [./nearest_node_number_2]
+    type = CSVDiff
+    input = nearest_node_number_2.i
+    csvdiff = nearest_node_number_2_out.csv
+    requirement = "The system shall have the capability to compute the nearest node number to a given point, and will choose the smallest node number if more than one node is equidistant."
+  [../]
+  [./nearest_node_number_2_2threads]
+    type = CSVDiff
+    input = nearest_node_number_2.i
+    csvdiff = nearest_node_number_2_out.csv
+    prereq = nearest_node_number_2
+    min_threads = 2
+    max_threads = 2
+    requirement = "The system shall have the capability to compute the nearest node number to a given point, and will choose the smallest node number if more than one node is equidistant, when the calculation employs threads."
+  [../]
+  [./nearest_node_number_2_2procs]
+    type = CSVDiff
+    input = nearest_node_number_2.i
+    csvdiff = nearest_node_number_2_out.csv
+    prereq = nearest_node_number_2_2threads
+    min_parallel = 2
+    max_parallel = 2
+    requirement = "The system shall have the capability to compute the nearest node number to a given point, and will choose the smallest node number if more than one node is equidistant, when the calculation employs multiple processors."
+  [../]
+  [./nearest_node_number_2_2procs_2threads]
+    type = CSVDiff
+    input = nearest_node_number_2.i
+    csvdiff = nearest_node_number_2_out.csv
+    prereq = nearest_node_number_2_2procs
+    min_parallel = 2
+    max_parallel = 2
+    min_threads = 2
+    max_threads = 2
+    requirement = "The system shall have the capability to compute the nearest node number to a given point, and will choose the smallest node number if more than one node is equidistant, when the calculation employs both threads and multiple processors."
+  [../]
```

Comment:
All of these tests can be removed. We run with these configurations already. In addition, the reqs aren't compliant with the SQA for the framework (some duplication)